lib/watir_drops/page_object.rb

```diff
@@ -18,52 +20,68 @@ def page_url
         end
       end
 
+      def page_title
+        define_method("page_title") do |args|
+          yield(*args)
+        end
+      end
+
       def element_list
         @element_list ||= []
       end
 
+      def required_element_list
+        @required_element_list ||= []
+      end
+
       def inherited(subclass)
         subclass.element_list = element_list.dup
+        subclass.required_element_list = required_element_list.dup
       end
 
-      def elements(name, &block)
+      def elements(name, required = false, &block)
```

Comment:
Yeah, it's pretty much the same thing as element just without the ability to have watir take an action by setting it equal to something. I like having it separate just to distinguish the element type that is being returned. Maybe it would be good to actually type enforce these. @tk8817 you have any preferences? Raise an exception if element doesn't return a `Watir::Element` and elements doesn't return a `Watir::ElementCollection`?